src/views/NoticesControl.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="notices-control">
+    <span class="notices-control__title">Информационные блоки</span>
+    <Notice v-for="notice in notices" :key="notice.title" v-bind="notice" />
```

Comment:
Wouldn't it be better to use `id` for `key`s? I think you cannot guarantee that where won't be 2 different entities with the same titles. Id is usually generated on the backend and usually it's unique across the DB.